lib/weather/presentation/widgets/weather_item.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:yet_another_weather_app/widgets/app_icons_icons.dart';
+
+import 'weather_attribute.dart';
+import 'weather_trait.dart';
+
+class WeatherItem extends StatelessWidget {
+  const WeatherItem({
+    super.key,
+  });
+
+  double translateImage(BuildContext context) {
+    final height = MediaQuery.of(context).size.height;
+    if (height < 700) {
+      return -32;
+    } else {
+      return 16;
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      alignment: Alignment.topCenter,
+      children: [
+        Transform.translate(
+          offset: const Offset(0, 45),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                width: 230,
+                height: 247,
+                padding: const EdgeInsets.fromLTRB(16, 40, 16, 16),
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(30),
+                  color: const Color(0xFFFBFBFB),
+                ),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      'Los Angeles, CA, USA',
+                      style: GoogleFonts.alegreyaSans(
+                        textStyle: const TextStyle(
+                          fontWeight: FontWeight.w700,
+                          fontSize: 20,
+                          height: 24 / 20,
```

Comment:
why 24 / 20 ? why not simply 1.2 in the whole file? I've found 12 / 10 or 16.8 / 14 but all of them are 1.2 actually.

Reply:
For the sake of convenience, I just use whatever value given in Figma inspect tab, not notifying they all resolve to same result. Will improve that in future.